osquery/events/windows/windowseventlogutils.h

```diff
@@ -0,0 +1,49 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed in accordance with the terms specified in
+ *  the LICENSE file found in the root directory of this source tree.
+ */
+
+#pragma once
+
+#include <ctime>
+
+#include <boost/algorithm/string.hpp>
+#include <boost/property_tree/json_parser.hpp>
+
+#include <osquery/flags.h>
+#include <osquery/utils/status/status.h>
+
+namespace osquery {
+class WindowsEventLog {
+ public:
+  struct Event final {
```

Comment:
Rename this to something like WindowsEventLogObject/WindowsEventLogObject/WELObject

Reply:
Instead of renaming the struct, I updated the alias name to `WELEvent` to avoid conflict. Please let me know your feedback on having the class `WindowsEventLog` to improve the code readability. I will update the name accordingly.

Reply:
The need to use the alias is because everything is still inside a class that is no longer needed; if you remove the class then there is no need to use type aliases